source/stylesheets/scss/furatto/_alerts.scss

```diff
@@ -0,0 +1,202 @@
+// $font-size - The alert base font-size. Default: $alert-font-size
+// $font-weight - The alert base font-weight. Default: $alert-font-weight
+//
+
+@mixin alert-base($padding: $alert-padding,
+                  $margin-bottom: $alert-margin-bottom,
+                  $font-size: $alert-font-size,
+                  $font-weight: $alert-font-weight) {
+  display: block;
+  font-size: $font-size;
+  font-weight: $font-weight;
+  margin-bottom: $margin-bottom;
+  padding: $padding;
+  position: relative;
+}
+
+
+//
+// @mixin
+// We use this to create the different styles of alerts
+// $background-color - The color for the alert background. Default: $alert-default-background
+// $text-color- The color for the alert text. Default: $alert-text-color
+//
+
+@mixin alert-style($background-color: $alert-default-background,
+                   $text-color: $alert-text-color) {
+
+  background: $background-color;
+  border: $alert-border-width $alert-border-style lighten($background-color, 10%);
+  color: $text-color;
+}
+
+//
+// @mixin
+// We use this to create the different styles of alerts
+// $radius - The radius for the alert variation. Default: $alert-radius
+// $box-shadow - The box shadow for the variation. Default: $alert-box-shadow
+//
+
+@mixin alert-variations($radius: $alert-radius,
+                       $box-shadow: $alert-box-shadow) {
+
+  &.radius {
+    @include border-radius($radius);
+  }
+
+  &.shadow {
+    @include box-shadow($box-shadow);
+  }
+}
+
+//
+// @mixin
+// We use mixin this to style links on the alerts
+// $color - The color for the anchors. Default: $alert-text-color
+// $font-weight - The font weight for links on the alert. Default: 600
+//
+
+@mixin alert-links($color: $alert-text-color,
+                   $font-weight: 600) {
+  a {
+    color: $color;
+    font-weight: $font-weight;
+    text-decoration: none;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+}
+
+//
+// @mixin
+// We use mixin this to style the close object for alerts
+// $font-size - The font-size for the alert close object. Default: $alert-close-font-size
+// $padding - The padding for the alert close object. Default: $alert-close-padding
+// $hover-opacity - The opacity when hovering the close object. Default: $alert-close-hover-opacity
+//
+
+@mixin alert-close($font-size: $alert-close-font-size,
+                   $padding: $alert-close-padding,
+                   $hover-opacity: $alert-close-hover-opacity,
+                   $hover-color: $alert-close-hover-color) {
+  .close {
+    background: none;
+    border: none;
+    cursor: pointer;
+    font-size: $font-size;
+    line-height: 0;
+    margin-top: -($font-size / 2);
+    padding: $padding;
+    position: absolute;
+    right: 0.5rem;
+    top: 50%;
+
+    &:hover,
+    &:focus {
+      @include opacity($hover-opacity);
+      color: $hover-color;
+      text-decoration: none;
+      text-shadow: 0 1px 0 #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.